src/Components/Notification.vue

```diff
@@ -84,7 +84,7 @@
 import axios from '@nextcloud/axios'
 import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
 import NcRichText from '@nextcloud/vue/dist/Components/NcRichText.js'
-import Close from 'vue-material-design-icons/Close.vue'
+import IconDelete from 'vue-material-design-icons/Delete.vue'
```

Comment:
That's a nope for me, it creates a way to heavy highlight on the action. Also any other so that has notifications that i can think of has an X there on each item, not a delete trash bin

Reply:
Rolled back. We shall reconsider initial issue then